charts/sck-otel/ci/o11y-values.yaml

```diff
@@ -14,3 +14,29 @@ splunkObservability:
   logsEnabled: true
   metricsEnabled: true
   tracesEnabled: true
+agent:
+  ports:
+    otlp:
+      containerPort: 4317
+      protocol: TCP
+      enabled: true
+    sfx-forwarder:
+      containerPort: 9080
+      protocol: TCP
+      enabled: true
+    zipkin:
+      containerPort: 9411
+      protocol: TCP
+      enabled: true
+    jaeger-thrift:
+      containerPort: 14268
+      protocol: TCP
+      enabled: true
+    jaeger-grpc:
+      containerPort: 14250
+      protocol: TCP
+      enabled: true
+    signalfx:
+      containerPort: 9943
+      protocol: TCP
+      enabled: true
```

Comment:
why explicitly add them here? it looks the same as default

Reply:
i removed hostPort. because it says no available port on host in the ci env.